extra/html/data_table.html

```diff
@@ -1,34 +1,26 @@
-<table class='table table-striped table-bordered' id='data-table'>
-  <thead>
-    <tr>
-      {% for item in data_table.first %}
-        <th>{{ item[0] }}</th>
-      {% endfor %}
-    </tr>
-  </thead>
-  <tbody id='data-table-body'>
+<div class="table-container">
+  <table class="table table-striped table-bordered" id="data-table">
+    <thead>
+      <tr>
+        {% for item in data_table.first %}
+          <th>{{ item[0] }}</th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody id="data-table-body">
 
-  </tbody>
-</table>
+    </tbody>
+  </table>
+</div>
 
-<div id='loading'>
+<div id="loading">
   <h1>Loading...</h1>
   <i class="fa fa-circle-o-notch fa-spin fa-fw"></i>
 </div>
 
+<script type="text/javascript" src="{{ site.github.url }}/js/papaparse.min.js"></script>
 <script>
-$(document).ready(function() {
-  var $table = $('table.table');
-  $table.floatThead({
-    scrollContainer: function($table) {
-      return $table.closest('.table-container');
-    }
-  });
-})
-</script>
-
-<script type='text/javascript' src='{{ site.github.url }}/js/papaparse.min.js'></script>
-<script>
+$(function() {
```

Comment:
It’s the one I always use. [According to this](http://learn.jquery.com/using-jquery-core/document-ready/), apparently that makes me an ‘experienced developer’!